src/engine/permission/Permission.ts

```diff
@@ -973,6 +979,28 @@ const validatePermissionMutationTypes = (
   }
 };
 
+const validatePermissionSubscriptionTypes = (
```

Comment:
I copied dumbly the check made for permission.mutation not sure it's needed for subscription.